packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -2304,7 +2327,7 @@ const tests = {
       errors: [
         "React Hook useEffect has a missing dependency: 'state'. " +
           'Either include it or remove the dependency array. ' +
-          `You can also write 'setState(state => ...)' ` +
+          `You can also write 'setState(s => ...)' ` +
```

Comment:
why did you decide to do a single character here?

Reply:
Showed it to people and they were confused seeing the same variable name because it wasn't obvious it's being shadowed. This explanation clicked tho.

Reply:
I guess I'm open to other changes, just want to avoid the shadowing. If people think this is confusing we can brainstorm more options.

Reply:
No I trust your judgement, I was just curious. It makes sense. Thanks!
(ashamed to say I usually use `x =>...`, which people around me hate)